.github/ISSUE_TEMPLATE/add_plugin.md

```diff
@@ -18,36 +18,34 @@ A brief description what your plugin does. Consider including screenshots to hel
 -->
 
 ## Checklist
 <!--
 Go through this checklist and confirm every item.
 
-It's fine if your plugin doesn't fulfill every item (e.g. it isn't production ready yet). You can still submit it!
-You can also still do code changes while the plugin is in review and fix issues on the fly.
+It's fine if your plugin doesn't fulfill every item (e.g. it isn't production ready yet). You can still submit it! You can also still do code changes while the plugin is in review and fix issues on the fly.
 
-Even if your plugin isn't production-ready, it might still be added to the Marketplace as "Beta".
-See https://developers.mattermost.com/extend/plugins/community-plugin-marketplace/#beta-plugins for more details.
+Even if your plugin isn't production-ready, it might still be added to the Marketplace as "Beta". See https://developers.mattermost.com/extend/plugins/community-plugin-marketplace/#beta-plugins for more details.
 
-If your plugin isn't production ready, please leave a comment stating whether you plan to fulfill the whole checklist or submit a "Beta" plugin.
+If your plugin isn't production-ready, please leave a comment stating whether you plan to fulfill the whole checklist or submit a "Beta" plugin.
 -->
 
 **Product requirements**
 
 - [ ] The plugin is published under an [Open Source license](https://opensource.org/licenses/alphabetical).
-- [ ] The source code is available in a public git repository.
+- [ ] The source code is available in a public Git repository.
 - [ ] There is a public issue or bug tracker for the plugin, which is linked in the plugin documentation and linked via `support_url` in the manifest.
 - [ ] The plugin provides detailed usage documentation with at least one screenshot of the plugin in action, list of features, and a development guide. This is typically a `README` file or a landing page on the web. The link to the documentation is set as `homepage_url` in the manifest. A great example is the [`README` of the GitHub plugin](https://github.com/mattermost/mattermost-plugin-github/blob/master/README.md).
 - [ ] For the current release and upcoming ones a changelog has to be published, with a link recorded in the `release_notes_url` property of the `plugin.json` manifest.
 - [ ] The plugin has to be out of Beta and be released with at least v1.0.0.
-- [ ] All configuration is accessible via the Mattermost UI.
-- [ ] The plugin ID defined in the manifest must not collide with the ID of an existing plugin in the Marketplace. It should follow [the documentation's suggested naming convention](https://developers.mattermost.com/extend/plugins/manifest-reference/#id).
+- [ ] All configuration is accessible via the Mattermost interface.
+- [ ] The plugin ID defined in the manifest must not collide with the ID of an existing plugin in the Plugin Marketplace. It should follow [the documentation's suggested naming convention](https://developers.mattermost.com/extend/plugins/manifest-reference/#id).
```

Comment:
Actually, the term used in the product is "Marketplace". https://github.com/mattermost/mattermost-developer-documentation/pull/747/ needs to get updated.